app/views/shared/_user_menu.html.erb

```diff
@@ -0,0 +1,64 @@
+<div
+    class="w-full border-t border-gray-200/50 pt-6 pb-2 pr-2 bg-black flex flex-col gap-2"
+    data-controller="click-toggle"
+    data-click-toggle-flippable-class="hidden"
+    >
+
+    <a href="#"
+       data-action="click->click-toggle#toggleClasses:prevent"
+       class="flex justfiy-left gap-2 rounded-lg text-white text-sm hover:bg-[#28282b] p-2"
+    >
+      <%= icon "cog" , size: 20 %>
+      Settings
+    </a>
+
+    <a href="/logout" class="flex justify-left gap-2 hover:bg-[#28282b] rounded-lg text-white text-sm p-2">
+    <%= heroicon "arrow-right-start-on-rectangle" , options: { class: 'h-5 w-5 text-gray-300' } %>
+    <p>Logout</p>
+  </a>
+
+  <div data-click-toggle-target="destination"
+       class="fixed inset-0 z-50 hidden"
+       >
+    <div
+      class="absolute inset-0 bg-[rgba(0,0,0,0.5)] flex justify-center items-center"
+      data-action="click->click-toggle#toggleClasses:self"
+    >
```

Comment:
stylistically I'm not sure I love the floating `>`, does it make it too hard to read the indentation of you include it at the end of the previous line? That's probably why you are floating it.

If we do float it, you probably intended to align with opening `<div`. The one right above this is aligned with `class`

Reply:
I don't like it either. I don't like breaking up the tags like this but I don't see a way around it. I think moving it up to the end of the data-action is fine. I put it here because it gets weird when you look at context with something nested inside the div, and then what level does that tag indent to? 

From my angle it does look to be aligned with the div though?

<img width="99" alt="image" src="https://github.com/hostedgpt/hostedgpt/assets/208647/ff5a42ed-1e85-41b2-a0d3-5123281328b0">